package.json

```diff
@@ -12,9 +12,10 @@
     "packages/groq",
     "packages/sanity"
   ],
+  "packageManager": "yarn@1.22.17",
```

Comment:
Settings this property makes `turbo` faster, and prevents someone from accidentally doing `npm install` on modern node.